packages/types/src/data-access-types.ts

```diff
@@ -20,6 +20,12 @@ export interface IDataAccess {
   ): Promise<IReturnGetChannelsByTopic>;
 }
 
+/** Enum of state possible for an action */
+export enum TransactionState {
```

Comment:
why is this type duplicated here and in transaction-types?

Reply:
also `ITimestampedTransaction`